app/src/main/java/com/bignerdranch/android/rickandmorty/retrofit/RickAndMortyService.kt

```diff
@@ -0,0 +1,27 @@
+package com.bignerdranch.android.rickandmorty.retrofit
+
+
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.create
+
+object RickAndMortyService {
+
+    private const val BASE_URL = "https://rickandmortyapi.com/api/"
+    private var retrofit: RickAndMortyAPI? = null
```

Comment:
можно предоставить лениво (будет инициализирован в момент вызова)
```suggestion
    private val retrofit: RickAndMortyAPI by lazy { provideRickAndMortyApi() }
```